custom_components/myskoda/device_tracker.py

```diff
@@ -68,17 +68,21 @@ def source_type(self) -> SourceType:  # noqa: D102
 
     @property
     def latitude(self) -> float | None:  # noqa: D102
-        position = self._vehicle_position()
-        if position is None:
-            return None
-        return position.gps_coordinates.latitude
+        if self.vehicle.is_capability_available(CapabilityId.PARKING_POSITION):
+            position = self._vehicle_position()
+            if position is None:
+                return None
+            return position.gps_coordinates.latitude
+        return None
```

Comment:
nit: `None` is the default return value at the end of a function so this line isn't needed.

Reply:
I suggest to move the
```python
if self.vehicle.is_capability_available(CapabilityId.PARKING_POSITION):
```

Check into the `_vehicle_position` method instead and return `None`.